src/to-have-display-value.js

```diff
@@ -38,7 +40,25 @@ export function toHaveDisplayValue(htmlElement, expectedValue) {
         `Expected element ${this.isNot ? 'not ' : ''}to have display value`,
         expectedValue,
         'Received',
-        value,
+        values,
       ),
   }
 }
+
+function getValues(tagName, htmlElement) {
+  return tagName === 'select'
+    ? Array.from(htmlElement)
+        .filter(option => option.selected)
+        .map(option => option.textContent)
+    : [htmlElement.value]
+}
+
+function getExpectedValues(expectedValue) {
+  return expectedValue instanceof Array ? expectedValue : [expectedValue]
+}
+
+function getQtyOfMatchesBetweenArrays(arrayBase, array) {
```

Comment:
I'd rather have a fully readable name like `getNumberOfMatchesBetweenArrays`. Same for the variables above that store this value.